modules/admin_manual/pages/configuration/files/encryption/encryption_configuration.adoc

```diff
@@ -54,7 +54,7 @@ connected to any external storage services, it is better to use other encryption
 
 [CAUTION]
 ====
-SSL terminates at or before the web server on the ownCloud server. Consequently, all files are in an unencrypted state between the SSL connection termination and the ownCloud code that encrypts and decrypts them. This is, potentially, exploitable by anyone with administrator access to your server. For more information, read: https://owncloud.org/blog/how-owncloud-uses-encryption-to-protect-your-data/[How ownCloud uses encryption to protect your data].
+SSL terminates at or before the web server on the ownCloud server. Consequently, all files are in an unencrypted state between the SSL connection termination and the ownCloud code that encrypts and decrypts them. This is, potentially, exploitable by anyone with administrator access to your server. For more information, read: https://owncloud.com/news/how-owncloud-uses-encryption-to-protect-your-data/[How ownCloud uses encryption to protect your data].
```

Comment:
```suggestion
SSL terminates at the same time or before the web server on the ownCloud server. Consequently, all files are in an unencrypted state between the SSL connection termination and the ownCloud code that encrypts and decrypts them. This is potentially exploitable by anyone with administrator access to your server. For more information, read: https://owncloud.com/news/how-owncloud-uses-encryption-to-protect-your-data/[How ownCloud uses encryption to protect your data].
```

Reply:
At least I think that what the "at or" in the first sentence was supposed to say.